packages/create-expo-app/src/__tests__/telemetry.test.ts

```diff
@@ -0,0 +1,221 @@
+import crypto from 'crypto';
```

Comment:
made a test for the telemetry client. i started with an e2e test, but since that is looking at built the out executable, we can't actually see any of the internals we want to test, so i switched to this. do you think this is sufficient, or should we try to add more tests?